onnxruntime/contrib_ops/cpu/activations.h

```diff
@@ -39,21 +39,33 @@ class Gelu : public OpKernel {
   Status Compute(OpKernelContext* context) const override {
     const auto* X = context->Input<Tensor>(0);
     Tensor* Y = context->Output(0, X->Shape());
-    concurrency::ThreadPool* tp = context->GetOperatorThreadPool();
-    const int64_t input_size = X->Shape().Size();
-    std::ptrdiff_t batch_size = static_cast<std::ptrdiff_t>(input_size);
-    //The cost comes from microbenchmark(manual tunning).
-    const double cost = 10.0;
-    const T* data = X->template Data<T>();
+    const T* input = X->template Data<T>();
     T* output = Y->template MutableData<T>();
-    concurrency::ThreadPool::TryParallelFor(tp, batch_size, cost, [data, output](ptrdiff_t first, ptrdiff_t last) {
-      ptrdiff_t len = last - first;
-      onnxruntime::ConstEigenVectorArrayMap<T> xm(data + first, len);
-      onnxruntime::EigenVectorArrayMap<T> ym(output + first, len);
-      ym = xm * static_cast<float>(M_SQRT1_2);
-      MlasComputeErf(output, output, len);
-      ym = xm * 0.5f * (ym + 1.0f);
-    });
+    concurrency::ThreadPool* tp = context->GetOperatorThreadPool();
```

Comment:
This basically reverts the previous set of changes to be in line with BiasGelu. Plus it was reported that it was throwing some accuracy errors.

Reply:
BiasGelu uses different logic (no Eigen any more). See [here](https://github.com/microsoft/onnxruntime/blob/4bdb5cc8e2e536dfae50a3159a5d71b07defecb3/onnxruntime/contrib_ops/cpu/bert/bias_gelu.cc#L48-L70) for FastGelu. It need slight change for Gelu.

Reply:
Sorry, I don't quite understand it.  What's wrong in my previous one, which changed?  I saw the single thread part is the same?